### ISSUE TITLE
Distributor horizontal pod autoscaler

### ISSUE THREAD

**Issue by kolesnikovae** [grafana/pyroscope#2445]
In order to protect distributors from OOM kills during traffic surges we should consider adding HPA to the helm chart. While this will not prevent OOM completely, it should reduce the blast radius.

**Comment by sepulworld** [comment]
Yes, would be worth the time to update the Pyroscope helm chart to match capabilities of the Tempo and Loki charts. Both of which support HPA out of the box for most of their components.
